source/tutorial.txt

```diff
@@ -6,46 +6,33 @@ Tutorials
 
 .. default-domain:: mongodb
 
+.. toctree::
+   :titlesonly:
+   :hidden:
+   
+   iOS with Swift </tutorial/ios-swift>	
+   Android with Kotlin </tutorial/java-sdk>	
+   Xamarin (iOS and Android) with C# </tutorial/dotnet>
+   React Native with JavaScript </tutorial/react-native>	
+   Web app with React, JavaScript, and GraphQL </tutorial/web-graphql>	
+   A Node.js Command Line Interface (CLI) </tutorial/nodejs-cli>
+
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Tutorials
----------
-
-Realm Database tutorials provide a walkthrough of the code required to
-implement Realm in mobile and backend applications. In these tutorials,
-you clone git repositories, and follow along with a tutorial that walks
-you through implementing Realm in your choice of:
-
-- Mobile application clients
-- Web browser
-- Node.js CLI
-
-If you prefer to learn by exploring and modifying a working application, 
-without walking through the code to implement Realm, check out the 
-:ref:`Realm template applications <template-applications>`.
-
-.. _task-tracker-tutorial:
-
-Task Tracker
-~~~~~~~~~~~~
-
-In this tutorial, you will build a collaborative task tracker app with
-Realm. 
+The Realm tutorials provide step-by-step guidance on building Realm apps. Each 
+of the tutorials starts with the 
+:ref:`Flexible Sync Template App <template-apps>` and then adds 
+functionality to it.  
 
-For more details, check out 
 
-- :doc:`What you will build <tutorial/preview>`
+.. button:: Check out the Template Apps
```

Comment:
So this button saying "Check out the Template Apps" right below a link for "Flexible Sync Template App" struck me as odd. I was slightly confused about which thing I should click. I like the idea of providing both the docs and a CTA button here but we should probably be more explicit about the link contents so users have some idea what they're getting when they click. Maybe the docs link could say something like "Learn more about the Flexible Sync Template App that this tutorial builds on" for the docs link, and then maybe "Create a Template App" or "Build a Template App" or something that's a slightly clearer CTA than "check out" for the button?